main.go

```diff
@@ -217,7 +217,7 @@ type dummyGAE struct {
 func configFromEnv(vargs *GAE, workspace *string) error {
 
 	// drone plugin input format du jour:
-	// http://readme.drone.io/plugins/plugin-parameters/
+	// https://docs.drone.io/plugins/overview/
```

Comment:
This should probably go to https://0-8-0.docs.drone.io/plugin-overview/ for now.